internal/inventory/asset.go

```diff
@@ -129,6 +134,9 @@ var (
 	AssetClassificationAwsRds                      = AssetClassification{Category: CategoryInfrastructure, SubCategory: SubCategoryDatabase, Type: TypeRelationalDatabase, SubType: SubTypeRDS}
 	AssetClassificationAwsS3Bucket                 = AssetClassification{Category: CategoryInfrastructure, SubCategory: SubCategoryStorage, Type: TypeObjectStorage, SubType: SubTypeS3}
 	AssetClassificationAwsSnsTopic                 = AssetClassification{Category: CategoryInfrastructure, SubCategory: SubCategoryMessaging, Type: TypeNotificationService, SubType: SubTypeSNSTopic}
+	// Azure
+	AssetClassificationAzureTenant       = AssetClassification{Category: CategoryInfrastructure, SubCategory: SubCategoryManagement, Type: TypeCloudAccount, SubType: SubTypeAzureTenant}
```

Comment:
I find management a super broad term. Is that something suggested by Tin? Maybe a quick ping on him would be cool

Reply:
Yeah, I'm pulling all classification names from [this spreadsheet](https://docs.google.com/spreadsheets/d/1TO4EtqTYcaW-yPCLcTVkp2PC97HYLwCUz6rvSAfwmtI/edit?gid=1300582090#gid=1300582090) word for word. We can talk on CDR sync maybe?